crates/sui-framework/packages/deepbook/tests/clob_tests.move

```diff
@@ -766,6 +768,114 @@ module deepbook::clob_test {
         end(test)
     }
 
+    fun test_clean_up_expired_orders_(test: Scenario): TransactionEffects {
+        let (alice, bob) = people();
+
+        // setup pool and custodian
+        let owner: address = @0xF;
+        next_tx(&mut test, owner);
+        {
+            clob::setup_test(0, 0, &mut test, owner);
+        };
+        next_tx(&mut test, alice);
+        {
+            mint_account_cap_transfer(alice, test::ctx(&mut test));
+        };
+        next_tx(&mut test, bob);
+        {
+            mint_account_cap_transfer(bob, test::ctx(&mut test));
+        };
+        next_tx(&mut test, alice);
+        {
+            let pool = test::take_shared<Pool<SUI, USD>>(&mut test);
+            let account_cap = test::take_from_address<AccountCap>(&test, alice);
+            let account_cap_user = account_owner(&account_cap);
+            let (base_custodian, quote_custodian) = clob::borrow_mut_custodian(&mut pool);
+            let alice_deposit_WSUI: u64 = 1000;
+            let alice_deposit_USDC: u64 = 110000;
+            custodian::test_increase_user_available_balance<SUI>(base_custodian, account_cap_user, alice_deposit_WSUI);
+            custodian::test_increase_user_available_balance<USD>(quote_custodian, account_cap_user, alice_deposit_USDC);
+
+            clob::test_inject_limit_order_with_expiration(&mut pool, CLIENT_ID_ALICE, 5 * FLOAT_SCALING, 500, 500, true,
+                CANCEL_OLDEST, 0, &account_cap, ctx(&mut test));
+            clob::test_inject_limit_order_with_expiration(&mut pool, CLIENT_ID_ALICE, 5 * FLOAT_SCALING, 500, 500, true,
+                CANCEL_OLDEST, 0, &account_cap, ctx(&mut test));
+            clob::test_inject_limit_order_with_expiration(&mut pool, CLIENT_ID_ALICE, 2 * FLOAT_SCALING, 1000, 1000, true,
+                CANCEL_OLDEST, 0, &account_cap, ctx(&mut test));
+            clob::test_inject_limit_order_with_expiration(&mut pool, CLIENT_ID_ALICE, 10 * FLOAT_SCALING, 10000, 10000, true,
+                CANCEL_OLDEST, 0, &account_cap, ctx(&mut test));
+            test::return_shared(pool);
+            test::return_to_address<AccountCap>(alice, account_cap);
+        };
+        next_tx(&mut test, bob);
+        {
+            let pool = test::take_shared<Pool<SUI, USD>>(&mut test);
+            let account_cap = test::take_from_address<AccountCap>(&test, bob);
+            let account_cap_user = account_owner(&account_cap);
+            let (base_custodian, quote_custodian) = clob::borrow_mut_custodian(&mut pool);
+            let alice_deposit_WSUI: u64 = 20000;
```

Comment:
this should be `bob_deposit_WSUI` and same for the line below.

Reply:
fixed.